newIDE/app/src/ObjectEditor/ObjectsEditorService.js

```diff
@@ -17,6 +17,9 @@ const ObjectsEditorService = {
       console.warn(
         `Object with type ${objectType} has no editor configuration registered. Please use registerEditorConfiguration to register your editor.`
       );
+      return this.getDefaultObjectJsImplementationPropertiesEditor({
```

Comment:
Isn't there any other check to do?

Reply:
Yes, I added a check on the existence of the EBO and only warn when it's not found.